src/bindings/c/include/openvino/c/ov_property.h

```diff
@@ -109,8 +109,8 @@ ov_property_key_cache_mode;
 
 /**
  * @brief Write-only property<ov_encryption_callbacks*> to set encryption/decryption function for model cache.
- * If ov_property_key_cache_encryption_callbacks is set, model cache will be encrypted/decrypted when saving/loading
- * model cache. ov_property_key_cache_encryption_callbacks is enabled in ov_core_compile_model_* only
+ * If ov_property_key_cache_encryption_callbacks is set, model topology in cache will be encrypted/decrypted when
+ * saving/loading model cache. This property is enabled in ov_core_compile_model_* only
```

Comment:
can we also update this document https://docs.openvino.ai/2024/openvino-workflow/running-inference/optimize-inference/optimizing-latency/model-caching-overview.html to reflect how to use this new feature?

Reply:
https://github.com/openvinotoolkit/openvino/pull/26432